tests/test_math.h

```diff
@@ -31,11 +31,54 @@
 #ifndef TEST_MATH_H
 #define TEST_MATH_H
 
-#include "core/os/main_loop.h"
+#include "core/math/transform.h"
+#include "core/math/math_funcs.h"
+#include "core/os/os.h"
+
+#include <math.h>
+#include <stdio.h>
+
+#include "tests/test_macros.h"
 
 namespace TestMath {
 
 MainLoop *test();
 }
 
+const float FM_PI = 3.1415926535897932384626433832795028841971693993751f;
```

Comment:
I think you can use `Math_PI` (defined in `core/math/math_defs.h`) instead of defining your own constant.